requirements/edx-sandbox/base.in

```diff
@@ -14,7 +14,7 @@ matplotlib==2.2.4                   # 2D plotting library
 numpy==1.7.2                        # Numeric array processing utilities; used by scipy
 pyparsing==2.2.0                    # Python Parsing module
 scipy==0.14.0                       # Math, science, and engineering library
-sympy==0.7.1                        # Symbolic math library
+sympy==0.7.2                        # Symbolic math library
```

Comment:
Are we sure that none of the [backwards compatibility breaks](https://github.com/sympy/sympy/wiki/Release-Notes-for-0.7.2#backwards-compatibility-breaks) are going to be a problem for existing code running in sandboxes?  I've been assuming that we're going to need different requirements files for Python 2 and Python 3 sandboxes; I think there are problems under 3.5 with the currently pinned versions of numpy and scipy also, noted [here](https://openedx.atlassian.net/wiki/spaces/AT/pages/939065888/edx-platform+Python+3+Discovery).

Reply:
I feel like we've discussed how to support jailed code once we go to Python 3, and maybe I didn't pay close enough attention. I don't see how we can just flip a switch on all the Python code embedded in courses.  Do we have a written and approved plan for what to do here?

Reply:
What I've been suggesting for this is essentially:

* Add support for running Python 3 codejails alongside the Python 2 ones
* Code that's tagged somehow as Python 3 compatible goes to the new codejails, everything else gets run in the old ones
* Work with course authors to gradually port their code to run in the new codejails
* Once we hit a critical mass of converted code, shut off the Python 2 codejails

There are summaries of this in a few Confluence docs, but not a detailed plan.  This is something we'll probably need to do again in the future, since several of the packages we install in codejails frequently make backwards-incompatible changes together with changes to support new Python versions.

Reply:
Yea, this definitely has some breaks but I was thinking we would just let the instructors know about them and give them some warning to update.  I think long term having a better upgrade process for sandboxes is good, I'm just not sure how much of that we have time for right now.  Is it a small lift to get multiple sandboxes?